tests/acceptance/test_organization_discover.py

```diff
@@ -1,33 +1,57 @@
 from __future__ import absolute_import
 
+from datetime import datetime, timedelta
+
 from sentry.testutils import AcceptanceTestCase
 
 
 class OrganizationDiscoverTest(AcceptanceTestCase):
```

Comment:
If you mixin SnubaTestCase the `store_event` method will go into snuba and the test will auto-skip should snuba not be running.